examples/multios_testing/sanity1.py

```diff
@@ -39,11 +39,11 @@ def testServerInside(self):
         self.run(" nohup nc -l %d > %s 2>&1 &" % (PORT, TFILE), shell=True, ignore_bg_processes=True)
         time.sleep(2)
         a = self.runHost("cat /etc/redhat-release").stdout.strip()
-        self.runHost("cat /etc/redhat-release | nc localhost %d" % PORT, shell=True)
+        self.runHost("Im Host | nc localhost %d" % PORT, shell=True)
```

Comment:
I'm testing difference between test host from host and guest from host. And in previous version there were dependency on host fedora version -> this leads to errors because version increased from 25->26 so this text is better.
This host from host test does not so big sense as guest from host, but it shows ability, that just run command is replaced by runHost and it is testing env vice-versa